MeshLib/MeshEditing/Mesh2MeshPropertyInterpolation.h

```diff
@@ -65,7 +61,7 @@ class Mesh2MeshPropertyInterpolation final
     void interpolateElementPropertiesToNodeProperties(std::vector<double> &interpolated_node_properties) const;
 
     Mesh const*const _src_mesh;
-    std::vector<double> const*const _src_properties;
+    std::string const& _property_name;
```

Comment:
The const reference  `_property_name` is initialized in the constructor and won't be updated in the life time of the object.